release/cloud_publish.template.json

```diff
@@ -2,7 +2,7 @@
   "steps": [
     {
       "name": "gcr.io/istio-testing/istio-builder:0.5.7",
-      "args": [ "-c", "repo init -u \"$_MFEST_URL\" -m \"$_MFEST_FILE\" -b \"$_MFEST_VER\" && repo sync" ],
+      "args": [ "-c", "git clone https://github.com/istio/istio -b \"$_BRANCH\" --depth 1 && istio/release/istio_checkout_code.sh -b \"$_BRANCH\" -r \"$_GCS_RELEASE_TOOLS_PATH\" " ],
```

Comment:
TODO
Consider using a consistent SHA for release scripts across multiple steps.